packages/core/src/command/rate-limiting/commands/get-settings.ts

```diff
@@ -0,0 +1,94 @@
+import {
+  errorToString,
+  MetriportError,
+  RateLimit,
+  rateLimitEntrySchema,
+  RateLimitOperation,
+} from "@metriport/shared";
+import { DocumentClient } from "aws-sdk/clients/dynamodb";
+import { DynamoDbUtils } from "../../../external/aws/dynamodb";
+import { capture, out } from "../../../util";
+import { Config } from "../../../util/config";
+import { createPrimaryKey, defaultOperationLimits } from "../shared";
+
+const region = Config.getAWSRegion();
+
+export async function getCxRateSettingValue({
+  cxId,
+  operation,
+  rateLimit,
+  create = true,
+  client,
+}: {
+  cxId: string;
+  operation: RateLimitOperation;
+  rateLimit: RateLimit;
+  create?: boolean;
+  client?: DocumentClient | undefined;
+}): Promise<number | undefined> {
+  const { log } = out(
+    `getCxRateSettingValue - cxId ${cxId} operation ${operation} rateLimit ${rateLimit}`
+  );
+  const settingsTableName = Config.getRateLimitingSettingsTableName();
+  if (!settingsTableName) return undefined;
+  const primaryKey = createPrimaryKey({ cxId, operation });
+  const ddbUtils = new DynamoDbUtils(region, settingsTableName, primaryKey, client);
+  try {
+    const rateLimits = await ddbUtils.getByKey({});
```

Comment:
Is this suppose to be an empty object?

Reply:
Yea, it was a one key parameter in retrospect, so the handler was like this:
```
getByKey({ sortKey }: { sortKey?: DdbMapping })
```

updated to just be this:
```
getByKey(sortKey: DdbMapping | undefined = undefined)
```

Reply:
No longer relevant.